### PR TITLE
Update SDK for new Azure Analysis Services API spec

### PR DESCRIPTION
**Pull request by michael-falk**
## Description

This PR is to update the .NET SDK according to a change in [Azure REST API specs](https://github.com/Azure/azure-rest-api-specs/pull/1515) for Azure Analysis Services. The corresponding API spec PR is [https://github.com/Azure/azure-rest-api-specs/pull/1515](https://github.com/Azure/azure-rest-api-specs/pull/1515). The changes were created using generate.cmd on the corresponding new swagger spec.
This change introduces fields for managing SKU capacity and the new querypoolConnectionMode property on customer servers.
This change updates tests to reflect the new additional properties.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by michael-falk** [comment]
PR for swagger spec change

https://github.com/Azure/azure-rest-api-specs/pull/1515

**Comment by shahabhijeet** [comment]
@michael-falk please add description about the PR.
As to what this PR is all about
What has been added or changed.
Any idea as to what kind of functionality been added/updated helps in reviewing the PR.

**Comment by michael-falk** [comment]
@shahabhijeet I've updated the description as per above request. However, I cannot remove the 'needs-revision' label. Would you please have a look?

File: src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj [hunk 1]
```diff
@@ -28,5 +28,11 @@
     <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\ScaleUpTest.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\ScaleOutTest.json">
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj]: @michael-falk please replace the individual .json files with something like below 
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj#L21

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/AnalysisServicesManagementClient.cs [hunk 2]
```diff
@@ -255,7 +247,7 @@ public AnalysisServicesManagementClient(System.Uri baseUri, ServiceClientCredent
         /// <exception cref="System.ArgumentNullException">
         /// Thrown when a required parameter is null
         /// </exception>
-        public AnalysisServicesManagementClient(System.Uri baseUri, ServiceClientCredentials credentials, HttpClientHandler rootHandler, params DelegatingHandler[] handlers) : this(rootHandler, handlers)
+        public AnalysisServicesManagementClient(System.Uri baseUri, Microsoft.Rest.ServiceClientCredentials credentials, System.Net.Http.HttpClientHandler rootHandler, params System.Net.Http.DelegatingHandler[] handlers) : this(rootHandler, handlers)
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/AnalysisServicesManagementClient.cs]: @michael-falk seems you are using an older version of autoRest.
Can you use the existing generate.cmd and if you need any info, use generate.cmd /? and you will get more information.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please use latest AutoRest by using Generate.cmd to regenerate.

**Comment by michael-falk** [comment]
@shahabhijeet I've rerun generate using the newest autorest

**Comment by michael-falk** [comment]
Closing to allow integration with other Analysis Services SDK changes (w/ Taiwu)

**Pull request closed without merging**